browser/brave_content_browser_client.cc

```diff
@@ -17,6 +17,7 @@
 #include "brave/browser/renderer_host/brave_navigation_ui_data.h"
 #include "brave/browser/tor/tor_profile_service_factory.h"
 #include "brave/common/brave_cookie_blocking.h"
+#include "brave/common/extensions/extension_constants.h"
```

Comment:
please put all extension related code in `#if BUILDFLAG(ENABLE_EXTENSIONS)` guards. The headers can go in the existing guard and the new code should be wrapped in `#if BUILDFLAG(ENABLE_EXTENSIONS)` as well